docker/docker-compose.yml

```diff
@@ -50,32 +49,6 @@ services:
       NUM_SHARDS: '1' # unused in testing, but required by vttestserver
       MYSQL_BIND_HOST: '0.0.0.0'
       FOREIGN_KEY_MODE: 'disallow'
-      VT_DIALECT: 'mysql80'
```

Comment:
I didn't find anything related to this param in vitess docs or GitHub so I assume it is safe to remove

Reply:
I think we originally copied it from https://github.com/planetscale/vitess-framework-testing where we got our first docker config for Vitess

Reply:
I believe I added it a few weeks back. It explicitly specifies that we consider Vitess as a flavor of `mysql:8+`. I believe it's the default option btw